src/components/AssetBreadcrumb/stories/AssetBreadcrumb.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import { Asset } from '@cognite/sdk';
+import { action } from '@storybook/addon-actions';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import {
+  ASSET_LIST_CHILD,
+  ASSET_ZERO_DEPTH_ARRAY,
+} from '../../../mocks/assetsListV2';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+import { AssetBreadcrumb } from '../AssetBreadcrumb';
+import customRendering from './custom-element-rendering.md';
+import fullDescription from './full.md';
+import handleCallbacks from './handle-callbacks.md';
+import maxLength from './max-length.md';
+
+class CogniteClient extends MockCogniteClient {
+  assets: any = {
+    retrieve: (ids: any) =>
+      new Promise(resolve => {
+        setTimeout(() => {
+          const allAssets = [...ASSET_ZERO_DEPTH_ARRAY, ...ASSET_LIST_CHILD];
+          const id = ids[0].id;
+          for (const asset of allAssets) {
+            if (asset.id === id) {
+              resolve([asset]);
+              return;
+            }
+          }
+        }, 1000);
```

Comment:
1 second here is really a lot.
that means 5 assets take 5 seconds
looks horribly slow in storybook
could you make it 100-200 instead?